BookingRoom UseCase/RoomRepositoryTest.java

```diff
@@ -1,5 +1,6 @@
 import org.junit.Test;
 
```

Comment:
The test method names are clear,
     but they can be even more descriptive and consistent in naming conventions.
     For example, instead of testInitializeRooms, you might consider something like testGetMultipleRoomsSuccessfully,
     making it clearer about exactly what is being tested
```suggestion
The test method names are clear,
     but they can be even more descriptive and consistent in naming conventions.
     For example, instead of testInitializeRooms, you might consider something like testGetMultipleRoomsSuccessfully,
     making it clearer about exactly what is being tested
```